content/sample-letters.fr.md

```diff
@@ -0,0 +1,77 @@
+{
+	"title": "Modèles de lettres pour les demandes RGPD",
+	"type": "page",
+	"aliases": ["letter-templates"]
+}
+
+Le RGPD vous accorde des droits étendus en ce qui concerne vos données personnelles. Afin de vous faciliter au maximum l'exercice de ces droits, nous avons préparé ces modèles de lettres.
+
+Avec eux, l'envoi d'une demande RGPD est simple : Tout ce que tu as à faire est d'entrer tes informations - nous t'avons fourni tout le reste.
+
+## Quel type de demande veux-tu envoyer ?
+
+Évidemment, il y a un modèle de lettre pour chaque type de demande. Nous prenons en charge les plus courantes.
+
+<article class="list-article icon-list-article">
+    <div class="col25 article-featured-image"><a href="/blog/sample-letter-gdpr-access-request/"><img class="image" src="/card-icons/view.svg" alt="Demande d'accès conformément à l'Art. 15 du RGPD"></a></div>
+
+    <div class="padded col75">
+        <a href="/blog/sample-letter-gdpr-access-request/"><h1>Demande d'accès conformément à l'Art. 15 du RGPD</h1></a>
+        
+        <p class="description">
+          Conformément à l'Art. 15 du RGPD, tu as un droit d'accès aux données personnelles qu'une entreprise stocke sur toi.
+        </p>
+    </div>
+    <div class="clearfix"></div>
+    <a class="button button-primary read-more-button" href="/blog/sample-letter-gdpr-access-request/">Voir le modèle de lettre&nbsp;<span class="icon icon-arrow-right"></span></a>
```

Comment:
We already have translations for some of these letters (in the [data repo](https://github.com/datenanfragen/data/tree/master/templates/fr)). It would be great to be able to link to them.

For now, we don't need to translate the whole page for the individual letters. I'll just create pages with just the sample letter text.

Reply:
I'll do than too